### ISSUE TITLE
OpenGL 2.0

### ISSUE THREAD

**Issue by xXSAMURAIXx** [zhuowei/Boardwalk#214]
Version of Boardwalk: 1.1

Version of Android: 4.4.2

Device model: LGUK14

Description of the problem: I can't use your app because It says I don't have OpenGL 2.0. Every time I open a world or multiplayer server I crash. Please help
![screenshot_2015-08-01-10-55-07](https://cloud.githubusercontent.com/assets/13602180/9022518/5be456da-383d-11e5-953d-eda5226243ce.png)

**Comment by RandomAltThing** [comment]
You are on 1.8.7. Don't use that version use 1.8.1

**Comment by xXSAMURAIXx** [comment]
Oh thanks

**Comment by xXSAMURAIXx** [comment]
Oh it didn't work. :(

**Comment by KStormTheOne** [comment]
Yes, 1.8 in any version crashes multiplayer and has no language and sounds, and reports a missing pack.mcmeta in options
